tests/test_boost.py

```diff
@@ -176,6 +176,10 @@ def test_obj_5D_strcat_intcat_rectangular_dak(use_weights):
         dhb.axis.Regular(9, -3.2, 3.2),
         storage=storage,
     )
+
+    # check that we are using the correct optimizer
+    assert h.__dask_optimize__ == dak.lib.optimize.all_optimizations
```

Comment:
Because dak is in the CI env; can we mock the case if it missing (config "awkward"=False)

Reply:
Conversely, it only matters that it is set correctly when dak is available. No need to test the other case (well, OK, until there are dask-histogram specific optimizations).